cdap-ui/app/cdap/components/PipelineDetails/PipelineDetailsTopPanel/PipelineDetailsDetailsActions/PipelineDetailsActionsButton/index.js

```diff
@@ -129,6 +130,18 @@ export default class PipelineDetailsActionsButton extends Component {
     );
   };
 
+  handlePipelineExport = () => {
+    if (window.sessionStorage.getItem('pipelineConfigTesting')) {
```

Comment:
This could be `Window.Cypress` check. That would help us confirm if we are running tests.

Reply:
Were you thinking the Window.Cypress check be in addition to (&&), instead of (completely replace current check), or an alternative to (||) looking for the token?

Reply:
I think we can replace this check with Window.Cypress check. Thats the pattern that we follow in other places.